src/main/kotlin/rasteplads/messageCache/MessageCache.kt

```diff
@@ -0,0 +1,34 @@
+package org.example.cache
+
+import java.time.LocalTime
+import java.util.*
+
+class MessageCache<T>(private val cacheTime: Long) {
+    private val cacheID: Queue<Pair<T, LocalTime>> = ArrayDeque()
+
+    fun cacheMessage(msg: T) {
+        if (!cacheID.any { it.first == msg })
+            cacheID.add(Pair(msg, LocalTime.now().plusMinutes(cacheTime)))
+
+        checkForOutdatedMessages()
+    }
+
+    private fun checkForOutdatedMessages() {
+        val msg = cacheID.iterator()
+
+        while (msg.next().second.compareTo(LocalTime.now()) ==
```

Comment:
Måske sku `LocalTIme.now()` gemmes i en var, bare for ik at recompute for meget? Ik så vigitigt igen